src/interpreters/interpreter_explain.rs

```diff
@@ -49,7 +49,7 @@ impl IInterpreter for ExplainInterpreter {
                 format!("{}", plan.display_graphviz())
             }
             DFExplainType::Pipeline => {
-                let pipeline = PipelineBuilder::create(self.ctx.clone(), plan).build()?;
+                let pipeline = PipelineBuilder::create(self.ctx.clone(), plan, true).build()?;
```

Comment:
Would it be better if we add a ctx.reset() here?
ctx.reset will reset all the variables about the current session